sklearn/svm/_classes.py

```diff
@@ -1037,6 +1039,12 @@ class NuSVC(BaseSVC):
 
     _impl = "nu_svc"
 
+    _parameter_constraints = {
+        **BaseSVC._parameter_constraints,  # type: ignore
+        "nu": [Interval(Real, 0.0, 1.0, closed="neither")],
```

Comment:
I would have expected this constraint to be the same than the one already defined in the base class. Do you see a reason why the bounds are not the same?

Reply:
I went back in the original paper. The upper bound is defined as:

```python
nu_max = 2 * min(n_samples_class_0, n_samples_class_1) / n_samples
```

so nu_max should be <= 1. This said, I tried in practise the problem on a balanced dataset and dual coefficients were not finite. I assume that we can still allow the parameter to be 1 but there might be a bug in the SVM algorithm that would need some investigation.

Reply:
This was an oversight. Fixing now.